upcase_booklist_app/data/author.py

```diff
@@ -0,0 +1,72 @@
+
+authors = [
+    {
+        "author_id": 1,
+        "name": "Eric Matthes",
+        "bio": "Eric Matthes is a high school math and science teacher living in Alaska who teaches an Introduction to Programming class in Python.",
+        "books": [
+            {
+                "book_id": 1,
+                "title": "Python Crash Course",
+                "summary": "A hands-on introduction to programming with Python.",
+                "image": "/static/img/python_crash_course.jpeg",
+            },
+        ],
+    },
+
+    {
+        "author_id": 2,
+        "name": "Naomi Ceder",
+        "bio": "Naomi Ceder is an American software developer, author, and conference speaker. She is the author of the second and third editions of The Quick Python Book.",
+        "books": [
+            {
+                "book_id": 1,
+                "title": "The Quick Python Book",
+                "summary": "A comprehensive guide to Python.",
+                "image": "/static/img/the_quick_python_book.jpeg",
+            },
+        ],
+    },
+    
+    {
+        "author_id": 3,
+        "name": "Aditya Bhargava",
+        "bio": "Aditya Bhargava is a Software Engineer with a dual background in Computer Science and Fine Arts.",
+        "books": [
+            {
+                "book_id": 1,
+                "title": "Grokking Algorithms",
+                "summary": "Learn how to apply common algorithms to practical programming problems.",
+                "image": "/static/img/grokking_algorithms.jpeg",
+            },
+        ],
+    },
+
+    {
+        "author_id": 4,
+        "name": "John Guttag",
+        "bio": "John Vogel Guttag is an American computer scientist, professor, and former head of the department of Electrical Engineering and Computer Science at MIT.",
+        "books": [
+            {
+                "book_id": 1,
+                "title": "Introduction to Computer Science and Programming Using Python",
+                "summary": "An introduction to programming and computational problem solving, using Python.",
+                "image": "/static/img/intro_to_computer_science.jpg",
+            },
+        ],
+    },
+
+    {
+        "author_id": 5,
+        "name": "Kristen Hunter",
+        "bio": "A seasoned software engineer, Kirsten has extensive experience in development, planning, customer support, and interaction.",
+        "books": [
+            {
+                "book_id": 1,
```

Comment:
nit-pick...the book ids are all 1, but they are all different. An individual book should have a distinct id.

Reply:
@paulghaddad Yes, but I'm assuming that authors can have more than 1 book. So I start with book 1, and then may add book 2, book 3, etc -- incrementing the books for that particular author.